actors/market/src/lib.rs

```diff
@@ -765,57 +625,24 @@ impl Actor {
         rt.transaction(|st: &mut State, rt| {
             let last_cron = st.last_cron;
             let mut updates_needed: BTreeMap<ChainEpoch, Vec<DealID>> = BTreeMap::new();
-            let mut msm = st.mutator(rt.store());
-            msm.with_deal_states(Permission::Write)
-                .with_locked_table(Permission::Write)
-                .with_escrow_table(Permission::Write)
-                .with_deals_by_epoch(Permission::Write)
-                .with_deal_proposals(Permission::Write)
-                .with_pending_proposals(Permission::Write)
-                .build()
-                .map_err(|e| {
-                    e.downcast_default(ExitCode::USR_ILLEGAL_STATE, "failed to load state")
-                })?;
+            let mut rm_cron_id: Vec<ChainEpoch> = vec![];
 
             for i in (last_cron + 1)..=rt.curr_epoch() {
                 // TODO specs-actors modifies msm as it's iterated through, which is memory unsafe
                 // for now the deal ids are being collected and then iterated on, which could
                 // cause a potential inconsistency in exit code returned if a deal_id fails
                 // to be pulled from storage where it wouldn't be triggered otherwise.
                 // Workaround a better solution (seperating msm or fixing go impl)
```

Comment:
Nit: delete this comment, which is now stale.

Reply:
Done 👍🏾